tools/cldr-apps/js/src/esm/cldrLoad.js

```diff
@@ -281,7 +281,9 @@ function verifyJson(json, subkey) {
   } else if (json.err_code) {
     var msg_fmt = cldrRetry.format(json, subkey);
     var loadingChunk = cldrDom.createChunk("", "p", "errCodeMsg");
-    loadingChunk.appendChild(cldrDom.createChunk("","i","glyphicon glyphicon-remove-sign fgred"));
+    loadingChunk.appendChild(
+      cldrDom.createChunk("", "i", "glyphicon glyphicon-remove-sign fgred")
+    );
```

Comment:
from an earlier PR. I've also applied this in another PR

Reply:
ok, thanks!